pallets/pallet-bank/src/tests.rs

```diff
@@ -421,3 +424,117 @@ fn pay_interest() {
 			assert!(Bank::check_total_issuance());
 		});
 }
+
+#[test]
+fn can_rotate_treasury() {
+	MockGenesisConfig::with_balances(vec![(ALICE, 100)]).build().execute_with(|| {
+		// Setup old treasury account and data
```

Comment:
after some thought, I think we should re-set the new total_issuance and check in the end.